tests/test_D2_model_constraints.py

```diff
@@ -55,6 +55,14 @@
     PATH_OUTPUT_FOLDER,
 )
 
+
+from multi_vector_simulator.D2_model_constraints import (
+    OEMOF_SOLPH_OBJECT_ASSET,
+    WEIGHTING_FACTOR_ENERGY_CARRIER,
+    RENEWABLE_SHARE_ASSET_FLOW,
+    OEMOF_SOLPH_OBJECT_BUS,
+)
```

Comment:
Do you know if this is necessary? In therory, you already imported `D2` in line 9. Not sure if then the variables would need to be called with `D2.OEMOF_SOLPH_OBJECT_ASSET` though, which of course would be more complicated (but also more explicit).